template.tpl

```diff
@@ -942,11 +942,18 @@ ___TEMPLATE_PARAMETERS___
     "type": "SELECT",
     "name": "bingEventType",
     "displayName": "Track Type",
+    "help": "Select 'Page View' only when your Freshpaint Bing Ads configuration has 'Automatically Send Page View Events' unchecked",
     "macrosInSelect": false,
+    "notSetText": "-",
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
```

Comment:
@ScottsoRobinson @ankurgupta1995 - added non-empty validator here, which is needed when notSetText is used to ensure an invalid bing tag can't be saved